.github/workflows/quality.yaml

```diff
@@ -0,0 +1,31 @@
+name: Prettier Formatting
+on:
+  pull_request:
+    branches:
+      - '**'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+jobs:
+  prettier:
+    name: Prettier
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        node-version: [18]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: npm/action-setup@v2
+        with:
+          version: 8
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+      - name: Install dependencies
+        run: npm install -D
+      - name: Check formatting with Prettier
+        run: npm run prettier:check:ci
+      - name: Format with Prettier
+        run: npm run prettier:format
```

Comment:
Does this also push the changes to the PR?

Reply:
I believe it does? It should run the prettier:format script